docs/topics/modules/planning_development_work.adoc

```diff
@@ -1,54 +1,29 @@
 [id="planning_development_work"]
 = Planning your development work
 
-When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow for launching and deploying a sample application before you launch the application.
+When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow before you launch the application.
 
-{osio} planner provides a set of predefined templates based on common development methodologies such as Agile and Scenario Driven Development. The development process you select while creating the space determines the planner's guided work item type hierarchy.
+{osio} planner provides a set of predefined templates based on common development methodologies such as Agile and Scenario Driven Development. The development process you select while creating the space determines the guided work item type hierarchy in the planner.
 
-This example walks you through the Scenario Driven Development planning process to plan your work.
+This example walks you through the Agile planning process to plan your work.
 
-. In the dashboard of your space, select the *Plan* tab at the top of the page to view the default *Backlog* view of the planner.
-. Select the *Show Tree* check box to see the work items and their children in a tree view. The tree view allows you to quickly add work items according to the guided hierarchy using the quick-add and in-line add options.
-. Use the quick-add option to create a *Scenario* as follows:
-.. In the *Add Work Item* pane, select *Scenario* from the drop-down list.
-+
-image::quickadd_scenario.png[Quick add scenario]
-+
-.. In the *Work Item Title* field, add *Enable an end to end, developer work-flow, using the continuous integration and deployment framework* as the title.
-.. Press kbd:[Enter] to create the scenario.
+. In the dashboard of your space, select the *Plan* tab at the top of the page. The default *Backlog* view displays the *Work Items* for your space.
+. Select the *Show Tree* check box to use the quick-add and in-line add options to quickly create workitems and their children.
+. In the *Add Work Item* pane, select *Theme* from the drop-down list, type *Enable a complete developer work-flow in a continuous integration and delivery framework* as the title, and press kbd:[Enter] to create the theme.
 
-. Create a child *Experience* for the *Scenario* using the in-line add option as follows:
-.. In the work item list, click the (+) icon adjacent to the scenario to see the in-line *Add Child Work Item* pane below it.
+. Create a child *Epic* for the *Theme* as follows:
```

Comment:
Again, here, the terminology is very confusing, IMO. Why are we calling this a child epic instead of a child work item? Also, child X implies that there is a parent X, but for some reason we're calling parent work items themes and child work items child epics. This is going to confuse users, I think.

Reply:
@mishaone have you selected Agile as the development process? All the WITs you are quoting are from SDD. Please test using an Agile space.

Reply:
Ah, right. Thanks, I can see the Theme and Epic now, but would still call them Work Items and Child Work Items for clarity and because the interface calls them the same for clarity as well. Referring to them by type will be confusing, like referring to JIRA issues as Features, Bugs, Tasks, etc.

Reply:
@mishaone Calling them workitem and child work item, imo increases ambiguity, user will wonder which child wit is being referred to, Child epic, grandchild story or great grandchild task. Also the realtion between the various types will never be clear.

Reply:
Honestly, I'm not sure why we need grandchild stories to begin with in a simple hello world application example. Maybe we should have a team meeting to discuss what exactly you want to show users in this example and whether all of it is relevant for a hello world example? I suspect simplifying things is better with this kind of use case.

Reply:
Exactly, this is what I suggested when I said let's meet to discuss the intent. We started off wanting to tie in the various seemingly disjoint actions in the GS as the intent, now we seem to be veering to having a single work item. 
A lot of discussions happened and efforts went in before this was pushed in the first time, both in google doc draft as well as the PR. But now we seem to want to change the intent. All good, except that we need to decide on how we move forward.